sdk/eventhub/event-hubs/src/eventHubConsumerClientModels.ts

```diff
@@ -1,45 +1,59 @@
 
-import { CloseReason, PartitionContext, EventProcessorOptions } from './eventProcessor';
+import { CloseReason, PartitionContext, EventProcessorOptions, EventProcessorBatchOptions } from './eventProcessor';
 import { PartitionCheckpointer } from './eventHubConsumerClient';
+import { ReceivedEventData } from '.';
```

Comment:
Nit: I know this works, and I don't _think_ it should break in any bundlers, but do you have any objection to putting the relative path to the file?

Reply:
I don't have any objection to swapping it back. The one reason I liked it (but "breaking bundlers" trumps "like") is that it makes it clear that I'm only using properly exported models.